src/docs/get-started/editor.md

```diff
@@ -16,7 +16,7 @@ even better experience. These plugins provide you with code completion, syntax
 highlighting, widget editing assists, run & debug support, and more.
 
 Follow the steps below to add an editor plugin for Android Studio,
-IntelliJ, or VS Code. If you want to use a different editor,
+IntelliJ, VS Code or Emacs. If you want to use a different editor,
```

Comment:
```suggestion
IntelliJ, VS Code, or Emacs. If you want to use a different editor,
```

Reply:
Are you sure this colon should exist? I think that in a sentence like that the last prefix to a noun should be an `or`, am I wrong?

Reply:
I'm not sure what you're referring to, but this seems like a case for serial commas: https://developers.google.com/style/commas

Reply:
Oh, that's new to me, sorry

Reply:
Done